net/dcwapd/files/dcwapd.init

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
+START=99
+STOP=01
+
+CONFIGURATION=dcwapd
+SCRIPTDIR=/etc/dcwapd
+
+start_dcwapd_service() {
+    procd_open_instance
+    procd_set_param command /bin/sh "${SCRIPTDIR}/start_dcwapd.sh"
```

Comment:
This pattern is not found anywhere else in the tree. Specifically, calling bin/sh into a script.

Would it be possible to merge start and stop_dcwapd.sh into the init script?

Reply:
There is actually quite a bit going on in `start_dcwapd.sh` and `stop_dcwapd.sh`, by way of the sourced `dcwapd.inc` file.  But I think that it could be all put into the single init script if that's preferable.  I'll play around with it.

One other thing that bugs me is the double `enable` that I have going on.  I understand that `procd` wants to automatically enable all services- is there a good way to mark the service as being initially disabled?  Ultimately the network should be configured before configuring and enabling Dual Channel WiFi.  From all of the testing that I've done, it seems to fail gracefully if things are misconfigured, but I don't like the idea of starting the service in an unknown/possibly bad state.  To work around this, I added an extra UCI `enabled` property that is disabled by default.  It doesn't seem like that is a standard thing to do, and it kind of bumps heads with procd.  In this case it might make sense to remove the extra UCI `enabled` property and let`dcwapd` start and fail.

Reply:
I merged all of the scripts into the single procd init script, and also fixed some things along the way :)

I removed the extra UCI `enabled` property, so when this PR is merged I'll also have to submit one against the LuCI app to remove it from the UI.

Reply:
$enabled in used in plenty of projects here, including transmission which I maintain.

Reply:
Ah ok, I see it now.  I guess if it's an accepted pattern then I'll put it back in.  It doesn't quite feel clean, but I do like how it prevents the `dcwapd` process from starting unless it is explicitly enabled (implying that it has also been configured).